docs/examples/example_lime.ipynb

```diff
@@ -359,7 +360,14 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "For guidelines on how to interpret such lime plots please see the [lime documentation](https://github.com/marcotcr/lime)."
+    "In these plots, the green bars signify that the corresponding feature referenced on the\n",
+    "y-axis, positively impacts CATE estimation. On the other side, the red bars, represent\n",
+    "features that negatively impact the CATE estimation.\n",
+    "Furthermore, the length of these coloured bars corresponds to the magnitude of each feature's\n",
+    "contribution towards the model prediction. Therefore, the longer the bar, the more\n",
+    "significant the impact of that feature on the overall model prediction.\n",
```

Comment:
```suggestion
    "significant the impact of that feature on the model prediction.\n",
```

Reply:
Overall might make it sound as though this was a global model interpretation, after all.